releases/24.09.0/24090 Release Notes.md

```diff
@@ -0,0 +1,469 @@
+
+## Simulation O3DE Gem Changes for 24.09.0 Release:
+
+**Major improvements**
+
+* ROS 2 Gem: added Georeference Component  
+* ROS 2 Gem: added possibility to spawn assets using WGS-84 data format in ROS2SpawnerComponent  
+* ROS 2 Gem: extended ROS2FrameComponent in Editor (split into Game and Editor components)  
+* ROS 2 RobotImporter: added Gazebo data parsing for camera, GNSS, IMU, and Lidar sensors  
+* ROS 2 RobotImporter: added Gazebo data parsing for skid steering and Ackermann drive models  
+* ROS 2 RobotImporter: added Gazebo data parsing for joint pose trajectory and joint state publisher configuration
+
+**Improvements**
+
+* ROS 2 Gem: added position control for joints (possible to set initial configuration; ROS 2 subscription and handler added)  
+* ROS 2 Gem: added camera stabilization (preventing tilting)  
+* ROS 2 Gem: added the option to transmit ROS 2 camera image using various image encodings  
+* ROS 2 Gem: modified clock designed  
+* ROS 2 RobotImporter: added Gazebo data parsing for topics, frames and namespaces  
+* ROS 2 RobotImporter: redesigned wizard
+
+**Fixes**
+
+* ROS 2 Gem: fixed ROS2SpawnerComponent  
+* ROS 2 Gem: fixed Lidar data publishing  
+* ROS 2 related Gems and project templates: cleaned-up versioning, updated to work with ROS 2 Jazzy  
+* ROS 2 related Gems and project templates: updated prefabs to match O3DE \>= 2310.0
+
+**Removed code**
+
+* ROS 2 Gem: removed outdated and duplicated documentation files from the Gem
+
+**Other Gems worth noticing:**
+
+* Pointcloud https://github.com/RobotecAI/robotec-o3de-tools/tree/main/Gems/Pointcloud  
+* ROS2ScriptIntegration https://github.com/RobotecAI/robotec-o3de-tools/tree/main/Gems/ROS2ScriptIntegration  
+* VehicleDynamics https://github.com/RobotecAI/o3de-vehicle-dynamics-gem
+
+## Rendering Specific Changes for 24.09.0 Release
+
+**Mobile Render Pipeline \-** Add support for an extremely performant mobile render pipeline built for tiled GPUs.
+
+* Optimised BRDF for mobile \- Added a new optimised BRDF for mobile as well as support for analytical Env IBL approximation.  
+* [Added Gobo (i.e texture projection) support for simple spot light.](https://github.com/o3de/o3de/commit/08e1d396a40b56db24ee26952da935c625d9def1)  
+* [Added CPU culling for decals](https://github.com/o3de/o3de/commit/6d23563d2b0633409969aeff29f9d10f12c6c875).  
+* [Added shadow support for SimpleSpotLights](https://github.com/o3de/o3de/commit/7ab669f974a6df543339a1cc625c34f6ad21b282).  
+* [Added support for decal colour and decal colour factor to help modulate final calculated decal colour.](https://github.com/o3de/o3de/commit/ed2f49bd2d56137e0ef88c9094dda1f328a8f401)  
+* [Added shader options for simple point/spot lights and decals](https://github.com/o3de/o3de/commit/c8ca92868bff7170bbf4e5d36154e7ed761bd89a).   
+* [Added support for cpu culling for simple-point and simple-punctual lights](https://github.com/o3de/o3de/commit/b6800d426a9e074c9f8a08dc6f9cdaf251ee95ca).  
```

Comment:
I would suggest removing the links to keep it consistent; I would suggest passing through a spellcheck to unify the language (`colour` -> `color`) in this example
```suggestion
* Added Gobo (i.e texture projection) support for simple spot light.
* Added CPU culling for decals
* Added shadow support for SimpleSpotLights
* Added support for decal color and decal color factor to help modulate final calculated decal color.
* Added shader options for simple point/spot lights and decals
* Added support for cpu culling for simple-point and simple-punctual lights
```